src/style.css

```diff
@@ -0,0 +1,135 @@
+* {
+  box-sizing: border-box;
+}
+
+h1,
+h2,
+h3,
+p,
+a,
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.page {
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: "Roboto", sans-serif;
+  color: #060b35;
+}
+
+.card__image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto;
+  margin-bottom: 40px;
+}
+
+.card__title {
+  width: 166px;
+  margin-bottom: 4px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__suptitle {
+  margin-bottom: 16px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
```

Comment:
https://codeguide.academy/html-css.html#css-order let's use this advice fix in all cases

Reply:
done